packages/kbn-utils/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@kbn/utils",
+  "main": "./target/index.js",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "kbn:bootstrap": "yarn build",
+    "kbn:watch": "yarn build --watch"
+  },
+  "dependencies": {
+    "@kbn/config-schema": "1.0.0",
+    "elastic-apm-node": "^3.7.0"
```

Comment:
I can't see you're using `elastic-apm-node` anywhere inside `@kbn/utils`?

Reply:
Good catch, that will be coming in a follow-up and should not be part of this. Removing.

Reply:
b4a0772f50e79ac261b0513a88ffe5491febf92a